<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddProject</class>
 <widget class="QDialog" name="AddProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#AddProject{
	background-image:url(img/addproject.jpg);
}
QTextEdit{
	background-color:rgb(235,199,255);
}  
     
QLineEdit{
	background-color:rgb(235,199,255);
}  
</string>
  </property>
  <widget class="QPushButton" name="BtnCancel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>350</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="proInfo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;黑体&quot;;
color:rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnConfirm">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>350</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;黑体&quot;;
color:rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>项目名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;黑体&quot;;
color:rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>项目客户：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;黑体&quot;;
color:rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>项目描述：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;黑体&quot;;
color:rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>项目经费：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>196</x>
     <y>60</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;黑体&quot;;
color:rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>项目经理：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>196</x>
     <y>90</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;黑体&quot;;
color:rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>项目状态：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;黑体&quot;;
color:rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>项目计划开始时间：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;黑体&quot;;
color:rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>项目计划结束时间：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="proName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>59</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="proClient">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="proManager">
   <property name="geometry">
    <rect>
     <x>266</x>
     <y>58</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="proState">
   <property name="geometry">
    <rect>
     <x>266</x>
     <y>88</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>未开始</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>进行中</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>已完成</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="proCost">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_91">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>194</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="planStartDay">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="planStartMonth">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_92">
   <property name="geometry">
    <rect>
     <x>283</x>
     <y>194</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="planStartYear">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>190</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="planEndYear">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>220</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="planEndMonth">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>220</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_93">
   <property name="geometry">
    <rect>
     <x>283</x>
     <y>224</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_94">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>224</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="planEndDay">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>220</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="proDescribe">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>260</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>155</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;黑体&quot;;
color:rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>项目位置：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="proAddress">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>155</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>196</x>
     <y>122</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;黑体&quot;;
color:rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>项目优先级：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="proPriority">
   <property name="geometry">
    <rect>
     <x>287</x>
     <y>118</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>最高级</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>次高级</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>高级</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>中级</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>低级</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>次低级</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>最低级</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
